src/testClientI.ts

```diff
@@ -0,0 +1,47 @@
+import {testRailConfig} from "./testRailConfig";
+// @ts-ignore
+import { sendHttpRequest } from '@qavajs/steps-api';
```

Comment:
It will require installing @qavajs/steps-api. I would suggest to describe sendHttpRequest in this module

Reply:
i thought about the same, but i see that we reuse some other modules, like memory

Reply:
as discussed in chat, lets eject sendHttpRequest into module?

Reply:
done